inc/class-main.php

```diff
@@ -41,6 +41,18 @@ public function init() {
 			add_filter( 'upload_mimes', array( $this, 'allow_meme_types' ) ); // phpcs:ignore WordPressVIPMinimum.Hooks.RestrictedHooks.upload_mimes
 			add_filter( 'wp_check_filetype_and_ext', array( $this, 'fix_mime_type_json_svg' ), 75, 4 );
 		}
+
+		add_filter(
```

Comment:
We can put this in the otter-blocks.php file because that's where we do rest of SDK stuff.